src/Models/BaseElement.php

```diff
@@ -457,6 +466,33 @@ public function Top()
         return (Controller::has_curr()) ? Controller::curr() : null;
     }
 
+    /**
+     * Determines whether this elemental block is indexable in search.
+     *
+     * By default, this uses the configurable variable is_indexable, but
+     * this method can be overridden to provide more complex logic if required.
+     *
+     * @return boolean
+     */
+    public function getIsIndexable(): bool
+    {
+        return (bool)$this->config()->get('is_indexable');
```

Comment:
Should this use the uninherited config so that inheritance wouldn't play role in excluding certain class from indexing?

Reply:
Ooh yes, that's a good idea. I'll make that adjustment.